lib/dry/core/equalizer.rb

```diff
@@ -137,6 +137,17 @@ def ==(other)
         end
       end
     end
+
+    # Build an equalizer module for the inclusion in other class
+    #
+    # ## Credits
+    #
+    # Equalizer has been originally imported from the equalizer gem created by Dan Kubb
+    #
+    # @api public
+    def self.Equalizer(*keys, **options)
+      Dry::Core::Equalizer.new(*keys, **options)
+    end
```

Comment:
When we originally imported dry-equalizer, we took the effort to define the `Dry.Equalizer` method in two places (`dry/core.rb` and `dry/core/equalizer.rb`) to  ensure legacy usage was covered, but completely overlooked that we needed to define a new `Dry::Core.Equalizer` method to match its new home and namespace!

Reply:
Good catch! However, this clashes with
https://github.com/dry-rb/dry-core/blob/c78caa5f1c4fc6535ce2f4a6c69c059bb1f353ae/lib/dry/core.rb#L33C1-L43C1
If you have `core/equalizer.rb` loaded first then loading `core.rb` will issue a warning about a redefined method. Adding `unless singleton_class.method_defined?(:Equalizer)` would solve this I think